SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+## Supported Versions
+
+Use this section to tell people about which versions of your project are
+currently being supported with security updates.
```

Comment:
This sounds a bit like placeholder text. You could change it to "Versions we currently support with security updates:"